scripts/remove/lidarr.sh

```diff
@@ -2,9 +2,13 @@
 user=$(cut -d: -f1 < /root/.master.info)
 systemctl disable --now -q lidarr
 
-rm -rf /home/$user/Lidarr
+rm -rf /opt/Lidarr
 rm -rf /home/$user/.config/Lidarr/
```

Comment:
I know we've been discussing the database removal; however this is inconsistent with our current sonarr/radarr removes. Perhaps we should not remove the database for now